CoreUtilities/src/au/gov/asd/tac/constellation/utilities/gui/field/framework/ConstellationInputListener.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2010-2024 Australian Signals Directorate
```

Comment:
2025. I'll stop mentioning these ones but will apply to the other subsequent new files as well